core/lib/libmbedtls/src/tee_lmd_provider.c

```diff
@@ -451,15 +612,28 @@ TEE_Result crypto_rng_add_entropy(const uint8_t *inbuf, size_t len)
 
 TEE_Result crypto_init(void)
 {
-	return TEE_ERROR_NOT_IMPLEMENTED;
+	return TEE_SUCCESS;
 }
 
-
 #if defined(CFG_CRYPTO_SHA256)
 TEE_Result hash_sha256_check(const uint8_t *hash, const uint8_t *data,
 		size_t data_size)
 {
-	return TEE_ERROR_NOT_IMPLEMENTED;
+	mbedtls_md_context_t hs;
+	uint8_t digest[TEE_SHA256_HASH_SIZE];
+
+	if (crypto_hash_init(&hs, TEE_ALG_SHA256) != TEE_SUCCESS)
```

Comment:
This implementation isn't useful for pager usage as it will make all hash implementations unpaged.

Reply:
Sorry for not think about it so careful here. I will to fix it by calling sha256 interfaces directly here.